postgres_lock/lock.py

```diff
@@ -56,8 +53,8 @@ def lock(self):
 
     def release(self):
         self.cursor.execute(
-            "SELECT {unlock_function}(%(lock_id)s)".format(unlock_function=self.unlock_function),
-            {"unlock_function": self.unlock_function, "lock_id": self.lock_id},
+            f"SELECT {self.unlock_function}(%(lock_id)s)",
```

Comment:
I know it's safe here - but would it be best practice (?) to use the [psycopg2.sql](https://www.psycopg.org/docs/sql.html) functions for string building?

```python
from psycopg2 import sql
```

```suggestion
            sql.SQL("SELECT {}(%(lock_id)s)").format(self.unlock_function)),
```

where `self.unlock_function` would be a `sql.Identifier()` rather than a plain string? 
?

Reply:
One to ponder on - seems like a good idea.

Only slight downside is that this package would then technically require psycopg2 or psycopg directly (which currently we don't!)